terraform/modules/application/logging.tf

```diff
@@ -3,7 +3,7 @@
 resource "aws_cloudwatch_log_group" "this" {
   #checkov:skip=CKV_AWS_158:TODO CLDC-2542 create and encrypt with a KMS key if retaining this log group
   #checkov:skip=CKV_AWS_338:TODO CLDC-2542 retaining logs for at least 1year greater than necessary for debugging ecs
-  name              = "${var.prefix}-app-ecs"
+  name              = "${var.prefix}-app"
   retention_in_days = 14

```

Comment:
I didn't change this to 60 (like in the main https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data-infrastructure/pull/60) because it's not being changed as part of the migration PR, and so there's no chance of a merge conflict. If I changed it to 60 here, and then in the main logging PR I changed it to something else, I'd need to remember to change it here.